utils.py

```diff
@@ -102,4 +102,40 @@ def get_model_coefficients(classifier, feature_set, covariate_names):
     coef_df['feature_set'] = feature_set
     
     
-    return coef_df
\ No newline at end of file
+    return coef_df
+
+def get_genes_coefficients(pca_object, classifier_object,
+                           expression_df):
+    """Identify gene coefficients from classifier after pca.
+
+    Args:
+        pca_object: The pca object from running pca on the expression_df.
+        classifier_object: The logistic regression classifier object.
+        expression_df: The original (pre-pca) expression data frame.
+
+    Returns:
+        gene_coefficients: A dataframe with entreze gene-ID, coefficient and
```

Comment:
I agree, including the gene name (symbol) would be good. 
@dhimmel any advice on how to do this... I saw that you did something similar in [notebook 3](https://github.com/cognoma/machine-learning/blob/master/3.pathway-classifier.ipynb) (see cell 41), but I'm not sure how.